andela/ah-fulldeck-frontend
# #162948958 Users should be able to create and view articles

#### What does this PR do?
Allows users to create and read articles

#### Description of Task to be completed?
- Users should view all articles
- Upon clicking a single article, the user should read the entire article contents
- As an Authenticated user, I should create/post an article

#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-fulldeck-frontend` on your local machine.
* Change your working directory to the project folder `cd ah-fulldeck-frontend`
* Checkout to the branch ` ft-crud-articles-162948958` using the command `git checkout ft-crud-articles-162948958`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start` and you should get the following message:
* Go to the link `/` to view all articles, click on single article to view it and `/createarticle` to create a new article

#### What are the relevant pivotal tracker stories?
[#162948958 Users should be able to create articles](https://www.pivotaltracker.com/story/show/162948958)

#### Screenshots 
#### Landing Page
<img width="1273" alt="screen shot 2019-02-15 at 01 45 39" src="https://user-images.githubusercontent.com/25298492/52822714-8d5dc700-30c3-11e9-9f48-e62ca4b0a9a9.png">

#### Single Article
<img width="1278" alt="screen shot 2019-02-15 at 01 45 57" src="https://user-images.githubusercontent.com/25298492/52822737-9f3f6a00-30c3-11e9-9465-0e3e0e2a9fe3.png">


#### Create Article
<img width="1274" alt="screen shot 2019-02-15 at 01 41 40" src="https://user-images.githubusercontent.com/25298492/52822757-b41bfd80-30c3-11e9-9d40-1c45203be0a4.png">
